docs/bin/dump_keywords.py

```diff
@@ -79,5 +81,4 @@
     # jinja2 < 2.10's indent filter indents blank lines.  Cleanup
     keyword_page = re.sub(' +\n', '\n', keyword_page)
 
-with open(outputname, 'w') as f:
-    f.write(keyword_page)
+update_file_if_different(outputname, keyword_page)
```

Comment:
You need to `keyword_page.encode('utf-8')` here (before, the file was opened in text mode).

Reply:
Fixed.